### PR TITLE
GPS fix's frame_id changed to body-fixed.

### PR DESCRIPTION
**Pull request by pavloblindnology**
According to [ROS](http://docs.ros.org/jade/api/sensor_msgs/html/msg/NavSatFix.html), frame_id of NavSatFix message is 
> This is a Euclidean frame relative to the vehicle, not a reference  ellipsoid.

So, usually, this should be `base_link`, not `map`.
Some ROS packages (like [robot_localization](http://wiki.ros.org/robot_localization)) rely on it to be specified exactly this way. 

This pull request adds child_frame_id parameter for global_position plugin and uses it for 

- global_position/global/header/frame_id
- global_position/raw/fix/header/frame_id
- global_position/local/child_frame_id (to make it complete)

### CODE REVIEW

File: mavros/src/plugins/global_position.cpp [hunk 1]
```diff
@@ -115,7 +116,8 @@ class GlobalPositionPlugin : public plugin::PluginBase {
 	ros::Subscriber gp_set_global_origin_sub;
 	ros::Subscriber hp_sub;
 
-	std::string frame_id;		//!< frame for topic headers
+	std::string frame_id;       //!< origin frame for topic headers
```
Comment (TSC21) [inline mavros/src/plugins/global_position.cpp]: Can you fix this indentation please?

File: mavros/src/plugins/global_position.cpp [hunk 2]
```diff
@@ -116,8 +116,8 @@ class GlobalPositionPlugin : public plugin::PluginBase {
 	ros::Subscriber gp_set_global_origin_sub;
 	ros::Subscriber hp_sub;
 
-	std::string frame_id;       //!< origin frame for topic headers
-	std::string child_frame_id; //!< body-fixed frame for topic headers
+	std::string frame_id;				//!< origin frame for topic headers
```
Comment (TSC21) [inline mavros/src/plugins/global_position.cpp]: Two less tabs and it will be good :)

Comment (pavloblindnology) [reply mavros/src/plugins/global_position.cpp]: What is the convention for Tab - 8 spaces?

Comment (TSC21) [reply mavros/src/plugins/global_position.cpp]: It's the current one being used. Also accepted by the `uncrustify` config.

Comment (pavloblindnology) [reply mavros/src/plugins/global_position.cpp]: Done.

**Comment by TSC21** [comment]
Thanks!

**Comment by TSC21** [comment]
@vooon Travis CI is not triggering at all...

**Pull request merged**